.github/workflows/build_addon.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: download releases files
-      uses: actions/download-artifact@v3
+      uses: actions/download-artifact@v4.1.7
```

Comment:
According to https://github.com/advisories/GHSA-cxww-7g56-2vh6:

> Patches
> Upgrade to version 4.1.7 or higher. Alternatively use 'v4' tag which points to the latest and secure version.

I prefer use the latest v4.

```suggestion
      uses: actions/download-artifact@v4
```